integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -754,6 +754,12 @@ public int utimensCallback(String path, long aSec, long aNsec, long mSec, long m
     return 0;
   }
 
+  @Override
+  public int symlink(String linkname, String path) {
+    LOG.debug("Ignore unsupported symlink operation, linkname {}, path{}", linkname, path);
```

Comment:
Do we want an unsupported operation symlink to return 0?
In any case (workloads) will unsupported operation symlink return 0 helps? 
Normally for an unsupported operation, we log WARN message and return error code.

Reply:
@LuQQiu done, please take a look~